### PR TITLE
dont let AI track pulsedemons

### LINKED ISSUE #33976: AI can track Pulse Demons with cameras
(WEB REPORT BY: west3436 REMOTE: 206.221.180.138:7777)
# Revision
eca84ad974cce4cc5a2753c461dae03ba1f3e854

# Description

# Steps to Reproduce
go to the tracking screen for AI and you can sometimes see Pulse Demon as an option

# What you Expected

# What Actually Happened

### PR DESCRIPTION
**Pull request by toomykins**
AI can no longer track pulse demons
fixes #33976
[bugfix]

### CODE REVIEW

**Comment by pedr69** [comment]
>get permabanned
>make bugfixes

**Comment by Eneocho** [comment]
> > get permabanned
> > make bugfixes

Based, dare I say.

File: code/game/machinery/camera/tracking.dm [hunk 1]
```diff
@@ -161,6 +161,8 @@
 			return FALSE
 		if(istype(target.loc, /obj/effect/dummy))
 			return FALSE
+		if(ispulsedemon(M))
```
Comment (pedr69) [inline code/game/machinery/camera/tracking.dm:164]: it is also my opinion that they should be trackable if the floor tile's been pried off

**Comment by nervere** [comment]
This is not the way to do it. Pulse demons can be visible with no floor tiles, and there's no reason an AI shouldn't be able to track it.

**Comment by xeroxemnas** [comment]
Troonykins moment

**Comment by Kurfursten** [comment]
lack of support

**Pull request closed without merging**